pages/docs/tooling/tooling-aste.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Artificial Solver Testing Environment (ASTE)
+permalink: tooling-aste.html
+keywords: tooling, aste
+---
+
+## Motivation
+
+ASTE is a tool which can be used to understand the dynamics of preCICE, how the data is goes in and out. It consist of C++ and Python modules.
+
+## Installation
+
+Please make sure that dependencies are installed.
+
+### C++ modules
+
+#### Required
+
+- preCICE
+- MPI
+- CMake
```

Comment:
Is MPI really a requirement or is it just needed for parallel execution? We should also add a link to the preCICE installation instructions. Also mention, that `cmake` and `mpi` are requirements of preCICE itself and most probably already satisfied.

Reply:
MPI is a requirement for ASTE but not for preCICE. IIRC, there is an option in preCICE to not using MPI just using TCP/IP communication

Reply:
>  MPI is a requirement for ASTE but not for preCICE. IIRC, there is an option in preCICE to not using MPI just using TCP/IP communication

Note that the socket communication is used for the intrasolver communication and in case your code (i.e. in this case aste) runs in parallel, mpi is a mandatory requirement for preCICE as well.